src/libstore/build.cc

```diff
@@ -4789,8 +4789,7 @@ void Worker::waitForInput()
        terminated. */
 
     bool useTimeout = false;
-    struct timeval timeout;
-    timeout.tv_usec = 0;
+    int timeout = 0;
```

Comment:
This should be `long` to be consistent with `std::max(1L, (long) ...)`.

Reply:
Fixed, thanks.